src/clj/rems/db/roles.clj

```diff
@@ -62,5 +62,6 @@
 
 (defn update-roles! [userid roles]
   (->> roles
+       (filter +db-roles+)
```

Comment:
Should we provide a migration that drops any wrong values from the DB table to clean it up?